lms/tasks/grading.py

```diff
@@ -0,0 +1,128 @@
+from sqlalchemy import select, exists
+
+from datetime import datetime
+from lms.models import (
+    GradingSyncGrade,
+    GradingSync,
+    LMSUser,
+    LTIRegistration,
+    Grouping,
+    ApplicationInstance,
+)
+from lms.services import LTIAHTTPService
+from lms.services.lti_grading.factory import LTI13GradingService
+from lms.tasks.celery import app
+
+
+@app.task()
+def sync_grades():
```

Comment:
`start_sync_grades`? 

Here we look for pending grade syncs and start them scheduled the individual grade sync jobs.